uisdk/build.gradle

```diff
@@ -57,7 +57,7 @@ afterEvaluate {
 
 dependencies {
 
-    api 'com.github.karhoo:karhoo-android-sdk:1.4.2'
+    api 'com.github.karhoo:karhoo-android-sdk:develop-SNAPSHOT'
```

Comment:
This should be for testing purproses only I think, should not be merged into develop.

Reply:
Hmm, it seems to be done on IOS at the moment where we have it set to our own develop directory, a branch, commit or version. But to get the new SDK changes properly, I would need to create a whole release version of the NSDK, which means each change we make on the NSDK = new release which I think doesn't really make much sense.

So I think its fine as long as the UISDK is never release with NSDK set to a branch outside of a version